content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Quality Gates
+kind: documentation
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+## Overview
+
+{{< callout url="#" header="false" >}}
+Quality Gates Rules are in beta.
+{{< /callout >}}
+
+Quality Gates allows you to gate your workflows based on signals in Datadog.
+For example, you can block a Pull Request from being merged if it introduces new flaky tests, even if the tests are passing.
+
+Using Quality Gates, you have control over what is merged into the default branch, and ultimately on what is deployed. ([AM] ADD SENTENCE)
+
+## Set up Quality Gates
+
+There are two main steps required to set up Quality Gates:
+1. Create one or more rules.
+2. Add the `datadog-ci gate evaluate` command in your CI pipeline.
```

Comment:
```suggestion
By using Quality Gates, you have control over what is merged into the default branch, and ultimately on what is deployed. ([AM] ADD SENTENCE)

Create one or more Quality Gates rule, and add the `datadog-ci gate evaluate` command in your CI pipeline.
```

Reply:
Do you think this is more clear for customers to understand over the "two main steps" thing?

Reply:
As this is a non-trivial feature to understand I would prefer to have a small setup section detailing that those are the only two steps required in order to set it up. if you think collapsing it into the main section is worth I can do it though. Feel free to unresolve the comment in case :)